src/main/java/com/exasol/sql/dql/select/Select.java

```diff
@@ -43,11 +46,40 @@ public Select all() {
      */
     public Select field(final String... names) {
         for (final String name : names) {
-            this.fields.add(new Field(this, name));
+            final DerivedColumn derivedColumn = new DerivedColumn(this, new Field(name));
+            this.derivedColumns.add(derivedColumn);
         }
         return this;
     }
 
+    /**
+     * Add a function.
+     * 
+     * @param functionName a name of function
+     * @param valueExpressions one or more value expression
+     * @return <code>this</code> instance for fluent programming
+     */
+    public Select function(final FunctionName functionName, final ValueExpression... valueExpressions) {
+        return function(functionName, "", valueExpressions);
+    }
+
+    /**
+     * Add a function.
+     *
+     * @param functionName a name of function
+     * @param valueExpressions one or more value expression
+     * @param derivedColumnName a name of a derived column
```

Comment:
Please check if we called that `derivedColumnName` `as` somewhere already.

Reply:
I saw `alias` somewhere, but it's not completely the same thing